firefox-beta.json

```diff
@@ -1,16 +1,18 @@
 {
+    "description": "Beta builds of Firefox: the popular open source web browser.",
     "homepage": "https://www.mozilla.org/en-US/firefox/beta/",
     "version": "nightly",
+    "license": "MPL-2.0",
     "architecture": {
         "64bit": {
-            "url": "https://download.mozilla.org/?product=firefox-beta-latest-ssl&os=win64&lang=en-US#/firefox-setup.exe"
+            "url": "https://download.mozilla.org/?product=firefox-beta-latest-ssl&os=win64&lang=en-US#/firefox.7z"
         },
         "32bit": {
-            "url": "https://download.mozilla.org/?product=firefox-beta-latest-ssl&os=win&lang=en-US#/firefox-setup.exe"
+            "url": "https://download.mozilla.org/?product=firefox-beta-latest-ssl&os=win&lang=en-US#/firefox.7z"
         }
     },
+    "extract_dir": "core",
     "bin": [
-        "firefox.exe",
```

Comment:
@r15ch13, I removed "firefox.exe" from `bin` for all manifests except [firefox.json](#diff-fed8398e0cf4638d316ac91d4ed9b614) to prevent conflicts if a user tries to install more than one build. Other builds are still available in `%PATH%` via their aliases (e.g. `firefox-beta`).